src/js/common/schemaform/widgets/SSNWidget.jsx

```diff
@@ -11,7 +11,11 @@ export default class SSNWidget extends React.Component {
     this.state = { val: props.value };
   }
   handleChange = (val) => {
-    const strippedSSN = val.replace(/[\- ]/g, '');
+    let strippedSSN;
+    if (val) {
+      strippedSSN = val.replace(/[\- ]/g, '');
+    }
+
     this.setState({ val }, () => {
       this.props.onChange(strippedSSN);
```

Comment:
Should add a unit test that shows calling `handleChange` with `undefined` results in `props.onChange` being called with `undefined` now.